packages/host/app/services/command-service.ts

```diff
@@ -0,0 +1,31 @@
+import Service, { service } from '@ember/service';
+import type OperatorModeStateService from '@cardstack/host/services/operator-mode-state-service';
+import { task } from 'ember-concurrency';
+import type MatrixService from '@cardstack/host/services/matrix-service';
+
+export default class CommandService extends Service {
+  @service declare operatorModeStateService: OperatorModeStateService;
+  @service private declare matrixService: MatrixService;
+
+  run = task(async (command: any, roomId: string) => {
```

Comment:
forgive my types here. Lets strengthen all the types when we add the ai bot change where it just returns a function tool call

Reply:
Oh this is actually just the CommandField

Reply:
how about a slightly stronger thing like: `Record<string, any>` since we know that at least the command is an object, not a string or a function, etc.